tests/validation/tests/v3_api/test_workload.py

```diff
@@ -858,3 +860,296 @@ def setup_project_by_role(role, remove_resource):
 
     else:
         return None, None, None, None
+
+# --------------------- rbac tests by workload type -----------------------
+
+@if_test_rbac
+@pytest.mark.parametrize("role", RBAC_ROLES)
+def test_rbac_wl_daemonset_create(role, remove_resource):
+    p_client, project, ns = setup_daemonset_project_by_role(role)
+    cluster = namespace["cluster"]
+    con = [{"name": "test1",
```

Comment:
Nit pick on this one, but this `con` container list variable is sometimes assigned in a single line and using two lines in other areas could we use one or the other for consistency? Unless there's no other way to make `flake8` pass.

Reply:
`con` containers are on one line. Unless they are too long for `flake8` like:
```
con = [{"name": "test1", "image": os.environ.get('RANCHER_TEST_IMAGE', "nginx")}]
```
Addressed in latest change.